sdk/src/client.ts

```diff
@@ -1,79 +1,32 @@
-import { WebSocket } from 'ws';
-import type { AuthStorage } from './composables/authentication.js';
-import { withoutTrailingSlash } from './utils.js';
-// import type { ReadItemsInput, ReadItemsOutput } from './commands/read-items.js';
-// import type { Command } from './types/index.js';
-
 export interface ClientConfig {
-	fetch?: fetch;
-	ws?: typeof WebSocket;
```

Comment:
I figured we should just rely on `globalThis.fetch` / `globalThis.ws`. In runtimes where those aren't set, it should be easy enough to polyfill them on the `globalThis` 🤔